src/components/Input/Input.js

```diff
@@ -138,7 +153,33 @@ class Input extends React.Component {
     }
   };
 
-  render() {
+  renderPasswordField = () => {
+    const toggleState = () => this.state._type === 'password' ? 'text' : 'password';
+    const handleClick = (evt) => {
+      evt.preventDefault();
+      this.setState({
+        _type: toggleState(this.state._type)
+      });
+    }
+
+    const getText = () => {
```

Comment:
This function name seems kind of ambiguous for an input.

Reply:
Yup, you're right, that was just laziness on my part. Fixed